modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-sagemaker.adoc

```diff
@@ -0,0 +1,207 @@
+pip install teradata_resultset_renderer_prebuilt-3.4.1.tar.gz
+pip install teradata_sqlhighlighter_prebuilt-3.4.1.tar.gz
+
+conda deactivate
+EOF
+----
+
+=== Create Jupyter Notebooks instance
+* On SageMaker navigate Notebooks, Notebook instances, create notebook instance
+* Choose a name for your notebook instance, define size (for demos the smaller available instance is enough)
+* Click in additional configurations and assign the recently created lifecycle configuration +
+
+image::{dir}/sagemaker-create-notebook-1.PNG[Create notebook instance,align="center" width=75%]
+* Click in additional configurations and assign the recently created lifecycle configuration
+* Assign the recently created IAM role to the notebook instance +
+
+image::{dir}/sagemaker-create-notebook-2.PNG[Assign IAM role to notebook instance,align="center" width=75%]
+
+* Paste the following link https://github.com/Teradata/lake-demos as the default github repository for the notebook instance +
+
+image::{dir}/sagemaker-create-notebook-3.PNG[Assign default repository for the notebook instance,align="center" width=75%]
+
+== Find the IP CIDR of your Jupyter Notebooks instance
+* Once the instance is running click on open JupyterLab + 
+
+image::{dir}/sagemaker-create-notebook-4.PNG[Initiate JupyterLab,align="center" width=75%]
+
+image::{dir}/sagemaker-create-loaded-env.PNG[Loaded JupyterLab,align="center" width=75%]
+
+* On JupyterLab open a notebook with Teradata Python kernel and run the following command for finding your notebook instance IP address.
+** We will whitelist this IP in your VantageCloud Lake environment in order to allow the connection.
+** This is for purposes of this guide and the notebooks demos. For production environments, a configuration of VPCs, Subnets and Security Groups might need to be configured and whitelisted.
+
+[source, python, role="content-editable"]
+---
+import requests
+def get_public_ip():
+    try:
+        response = requests.get('https://api.ipify.org')
+        return response.text
+    except requests.RequestException as e:
+        return "Error: " + str(e)
+my_public_ip = get_public_ip()
+print("My Public IP is:", my_public_ip)
+---
+
+== VantageCloud Lake Configuration
+* In the VantageCloud Lake environment, under settings, add the IP of your notebook instance +
+
+image::{dir}/sagemaker-lake.PNG[Initiate JupyterLab,align="center" width=75%]
+
+== Jupyter Notebook Demos for VantageCloud Lake
+
+=== Configurations
+* The file https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file] should be edited to add the required credentials to run the demos +
+
+image::{dir}/sagemaker-vars.PNG[Initiate JupyterLab,align="center" width=75%]
+
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+
+| *"host"* 
+| Public IP value from your VantageCloud Lake environment
+
+| *"UES_URI"* 
+| Open Analytics from your VantageCloud Lake environment
+| *"dbc"*
+| The master password of your VantageCloud Lake environment.
+|====
```

Comment:
This table should be indented with the bullet point

Reply:
Solved:
![image](https://github.com/Teradata/quickstarts/assets/29882294/e3097660-f8cc-4929-9d59-41083b6e80f1)